x/bank/simulation/operations_test.go

```diff
@@ -121,6 +123,63 @@ func (suite *SimTestSuite) TestSimulateMsgMultiSend() {
 	suite.Require().Len(futureOperations, 0)
 }
 
+func (suite *SimTestSuite) TestSimulateModuleAccountMsgSend() {
+	const (
+		moduleAccCount = 2
+		accCount       = 1
+	)
+	accounts := suite.setupAccounts(moduleAccCount, accCount)
+
+	// begin a new block
+	suite.app.BeginBlock(abci.RequestBeginBlock{Header: tmproto.Header{Height: suite.app.LastBlockHeight() + 1, AppHash: suite.app.LastCommitID().Hash}})
+
+	// execute operation
+	op := simulation.SimulateMsgSendToModuleAccount(suite.app.AccountKeeper, suite.app.BankKeeper)
+
+	s := rand.NewSource(1)
+	r := rand.New(s)
+
+	operationMsg, futureOperations, err := op(r, suite.app.BaseApp, suite.ctx, accounts, "")
+	suite.Require().Error(err)
+
+	var msg types.MsgSend
+	types.ModuleCdc.UnmarshalJSON(operationMsg.Msg, &msg)
+
+	suite.Require().False(operationMsg.OK)
+	suite.Require().Equal(operationMsg.Comment, "invalid transfers")
+	suite.Require().Equal(types.TypeMsgSend, msg.Type())
+	suite.Require().Equal(types.ModuleName, msg.Route())
+	suite.Require().Len(futureOperations, 0)
```

Comment:
If you declare `require := suite.Require()` in the top of the function, then this is shorter and easier to read.